api/utils/requestProcessor.js

```diff
@@ -0,0 +1,1159 @@
+const Promise = require('bluebird');
+const common = require('./common.js');
+const {validateUser, validateUserForRead, validateUserForWrite, validateGlobalAdmin} = require('./rights.js');
+const authorize = require('./authorizer.js');
+const plugins = require('../../plugins/pluginManager.js');
+const versionInfo = require('../../frontend/express/version.info');
+const log = require('./log.js')('core:api');
+const validateUserForWriteAPI = validateUser;
+const validateUserForDataReadAPI = validateUserForRead;
+const validateUserForDataWriteAPI = validateUserForWrite;
+const validateUserForGlobalAdmin = validateGlobalAdmin;
+const validateUserForMgmtReadAPI = validateUser;
+
+const countlyApi = {
+    data: {
+        usage: require('../parts/data/usage.js'),
+        fetch: require('../parts/data/fetch.js'),
+        events: require('../parts/data/events.js'),
+        exports: require('../parts/data/exports.js')
+    },
+    mgmt: {
+        users: require('../parts/mgmt/users.js'),
+        apps: require('../parts/mgmt/apps.js')
+    }
+};
+
+/**
+ * Process Request
+ * @param params
+ * @returns {boolean}
+ */
+const processRequest = (params) => {
```

Comment:
Can we document this module a JSDoc compatible to be included in the docs, including proper module name and exmaples, etc.

Just like this for example
https://github.com/Countly/countly-server/blob/master/api/utils/taskmanager.js